splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/UpgradeUtils.java

```diff
@@ -50,6 +50,8 @@ public static void initializeConglomerateSITable(TransactionController tc) throw
         SIDriver driver = SIDriver.driver();
         int rowsRewritten = 0;
 
+        int batchSize = 100;
```

Comment:
Maybe we could use `conf.getMaxBufferEntries()`, what we use for the write pipeline. This defaults to 1000 rows.